### PR TITLE
delete tickets

### PR DESCRIPTION
**Pull request by tejaswinichile**
Issue #373

### CODE REVIEW

**Comment by tejaswinichile** [comment]
@sudeeptarlekar _a

**Comment by sudeeptarlekar** [comment]
@tejaswinichile _a please rebase master

File: app/graphql/mutations/ticket_mutations.rb [hunk 1]
```diff
@@ -31,4 +31,28 @@ class Mutations::TicketMutations
       }
     }
   end
+
+  Delete = GraphQL::Relay::Mutation.define do
+    name "DeleteTicket"
+
+    input_field :id, types.String
```
Comment (sudeeptarlekar) [inline app/graphql/mutations/ticket_mutations.rb]: isn't ID is required parameter in this mutation?

**Comment by tejaswinichile** [comment]
@sudeeptarlekar _a Please check.

**Pull request merged**